docs/v1/details/environment-variables.mdx

```diff
@@ -1,19 +1,24 @@
 ---
 title: "Environment Variables"
 description: "Environment Variables that AgentOps will look for"
+mode: "wide"
 ---
 
 ### Sample .env file
 
-```.env
+```python .env
```

Comment:
is this intentional? its not a python code block

Reply:
Yes it's just to give it highlighting. Was just a black/white blob before. End user doesn't see anything about python